lib/data/repositories/plot_repository.dart

```diff
@@ -16,19 +18,37 @@ class PlotRepository{
     _firestoreManager = FireStoreManager.get();
   }
 
-  Future<List<CropEntity>> getListOfCrops() {
+  Future<List<CropEntity>> _getListOfCrops() {
     return _firestoreManager.getCrops();
   }
 
-  /// Adds stages to the supplied crop entities
-  Future<dynamic> getListOfCropStages(List<CropEntity> cropsWithoutStages) {
-    return _firestoreManager.getStages(cropsWithoutStages);
+  Future<dynamic> _getListOfCropStagesForCrop(CropEntity cropEntity) {
+    return _firestoreManager.getStagesForCrop(cropEntity);
   }
 
   /// Adds images to the supplied crop entities
-  Future<dynamic> getListOfCropsWithImages(List<CropEntity> cropsWithoutImages) {
+  Future<dynamic> _getListOfCropsWithImages(
+      List<CropEntity> cropsWithoutImages) {
     return _firestoreManager.getCropsImagePath(cropsWithoutImages);
   }
 
+  Future<List<CropEntity>> getArticles() async {
+    Stopwatch sw = new Stopwatch();
+    sw.start();
+    List<CropEntity> listOfCrops = await _getListOfCrops();
+    debugPrint('getListOfCrops() ${sw.elapsed.inMilliseconds} ms ');
```

Comment:
debug prints

Reply:
Agree. For beta1 though I think these are safe to go in.

Reply:
more concerned about the stopwatch assuming it runs outside of the debug build, however the using of printing should be made clear in the standards guide if not already @albertmiro